README.md

```diff
@@ -1,133 +1,160 @@
+- The CONFLUENT_KEY and CONFLUENT_SECRET environment variables are used to authenticate with the bootstrap server.
+
+### Values
+- DOCKER_HOST_IP must be set to the bootstrap server address (excluding the port)
+- KAFKA_TYPE must be set to "CONFLUENT"
+- CONFLUENT_KEY must be set to the API key being utilized for CC
+- CONFLUENT_SECRET must be set to the API secret being utilized for CC
+
+## CC Docker Compose File
+There is a provided docker-compose file (docker-compose-confluent-cloud.yml) that passes the above environment variables into the container that gets created. Further, this file doesn't spin up a local kafka instance since it is not required.
+## Release Notes
+The current version and release history of the Jpo-s3-deposit: [Jpo-s3-deposit Release Notes](<docs/Release_notes.md>)
+
+## Note
+This has only been tested with Confluent Cloud but technically all SASL authenticated Kafka brokers can be reached using this method.
+
+# Run Script
+The run.sh script can be utilized to run the PPM manually.
+
+It should be noted that this script must be run from the project root folder, or it will not work.
+
+# Docker Compose Files
+The docker-compose.yml file can be used to spin up the PPM as a container, along with instances of kafka and zookeeper.
+
+The docker-compose-confluent-cloud.yml file can be used to spin up the PPM as a container by itself. This depends on an instance of kafka hosted by Confluent Cloud.
+
+# Launch Configurations
+A launch.json file with some launch configurations have been included to allow developers to debug the project in VSCode.
+
+The values between braces < > are stand-in and need to be replaced by the developer. 
+
+To run the project through the launch configuration and start debugging, the developer can navigate to the Run panel (View->Run or Ctrl+Shift+D), select the configuration at the top, and click the green arrow or press F5 to begin.
+
+# MongoDB Deposit Service
+The mongo-connector service connects to specified Kafka topics (as defined in the mongo-connector/connect_start.sh script) and deposits these messages to separate collections in the MongoDB Database. The codebase that provides this functionality comes from Confluent using their community licensed [cp-kafka-connect image](https://hub.docker.com/r/confluentinc/cp-kafka-connect). Documentation for this image can be found [here](https://docs.confluent.io/platform/current/connect/index.html#what-is-kafka-connect).
+
+## Configuration
+Provided in the mongo-connector directory is a sample configuration shell script ([sample_connect_start.sh](./mongo-connector/sample_connect_start.sh)) that can be used to create kafka connectors to MongoDB. For testing please create a copy of this file and rename it to `connect_start.sh` in the same file location. To configure more sinks to be created, use the following format:

```

Comment:
Let's add a note saying this format will be appended to the beginning of the connect_start.sh file.